packages/cli/src/zostso/issue/command/Command.definition.ts

```diff
@@ -37,7 +37,15 @@ export const CommandDefinition: ICommandDefinition = {
             name: "suppress-startup-messages",
             aliases: ["ssm"],
             type: "boolean",
-            description: "Suppress console messages from start of address space."
+            description: "Suppress console messages from start of address space.",
+            defaultValue: true
+        },
+        {
+            name: "stateful",
+            aliases: ["sf"],
+            type: "boolean",
+            description: "Statefulness of address space created for TSO Command.",
+            defaultValue: false
```

Comment:
This is the crucial thing that we should document.

Users can explicitly ask for the old behavior if they specify `--ssm false`.

Otherwise, we will try the new behavior (if available) and fallback to the old behavior if the API is not available.

Also, if `--ssm false`, the value provided for `--stateful` won't really matter.

@anaxceron,
Could you help us here?